script.js

```diff
@@ -0,0 +1,80 @@
+//sets isO to false so first click is an X, next click an O, then an X, etc.
+var isO = false;
```

Comment:
I would try to find a way to move these global variables into another function or object